packages/edit-post/src/test/editor.native.js

```diff
@@ -56,6 +61,26 @@ describe( 'Editor', () => {
 			RNReactNativeGutenbergBridge.editorDidMount
 		).toHaveBeenCalledWith( [ 'core/notablock' ] );
 	} );
+
+	it( 'toggles the editor from Visual to HTML mode', async () => {
+		// Arrange
+		let toggleMode;
+		subscribeParentToggleHTMLMode.mockImplementation( ( callback ) => {
+			toggleMode = callback;
+		} );
+		const screen = await initializeEditor();
+		await addBlock( screen, 'Paragraph' );
+
+		// Act
+		const paragraphBlock = getBlock( screen, 'Paragraph' );
+		fireEvent.press( paragraphBlock );
+
+		toggleMode();
+
+		// Assert
+		const htmlEditor = await screen.findByLabelText( 'html-view-content' );
+		expect( htmlEditor ).toBeDefined();
```

Comment:
I recognize this is a WIP draft, but prematurely dropping a suggestion to use the [`toBeVisible`](https://github.com/WordPress/gutenberg/blob/ce4dd6167517ed52de1a120ee4015942a7f73513/test/native/matchers/to-be-visible.js#L44) helper, if it is compatible. 🙇🏻

Reply:
Thanks for the suggestion! Updated in https://github.com/WordPress/gutenberg/pull/51650/commits/a5f61bb9df70c54dc214be00a30fa3f75ccdf9d1